docker/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Python container for running the main script and generating postscript -------
+FROM python:3.12-slim AS paper-toaster
+WORKDIR /app
+ENV WORK_DIR=/workdir
+
+RUN adduser \
+    --disabled-password \
+    --gecos "The master of toast" \
+    --uid 1000 toastmaster
+USER toastmaster
+ENTRYPOINT ["python", "-m", "papertoaster"]
+
+COPY src /app
+
+# Linux container with GhostScript for converting PostScript to PDF/PNG --------
+FROM alpine:3.20 AS post-toast-ghost
+
+WORKDIR /scripts
+ENTRYPOINT ["sh", "./convert_receipt.sh"]
```

Comment:
While I doubt this command will change, it's more likely to change than the ghostscript dependency, so let's move this after setting the user